pkg/build/client/clients.go

```diff
@@ -0,0 +1,69 @@
+package client
```

Comment:
so as soon as we add a second file to this package, the name would be ok, right?